manim/mobject/text/tex_mobject.py

```diff
@@ -97,24 +101,24 @@ def __init__(
         self.init_colors()
 
         # used for scaling via font_size.setter
-        self.initial_height = self.height
+        self.initial_height: float = self.height
 
         if height is None:
-            self.font_size = self._font_size
+            self.font_size: float = self._font_size
```

Comment:
I don't think we need to typehint property setters?

Reply:
Right - when I typed `self.font_size`, I forgot that it was a property.